timesketch/lib/analyzers/interface.py

```diff
@@ -432,6 +436,73 @@ def __init__(self, index_name, sketch_id):
         self.sketch = Sketch(sketch_id=sketch_id)
         super(BaseSketchAnalyzer, self).__init__(index_name)
 
+    def event_pandas(
```

Comment:
This looks good, but the method looks very similar to event_stream(). Could we refactor this into a parameter on event_stream() instead, with a bool to return pandas or the generator?

something like:
```event_stream(...., return_pandas=False)```

WDYT

Reply:
spoke offline, mainly due to the fact that event_stream is a generator but pandas is a single object that gets returned, and I don't think we should be mixing those (pylint will be very unhappy)... makes the API confusing

Reply:
I Also thought about just calling the event_stream and create a list there, that is also an option, but it also has some overhead there, some things are done slightly differently in this function than in the event stream one.

Reply:
Ack, I agree.